sonar-project.properties

```diff
@@ -0,0 +1,10 @@
+sonar-project.properties
+sonar.organization=fedora-infra
+sonar.projectKey=fedora-infra/noggin
+sonar.sources=.
+sonar.host.url=https://sonarcloud.io
+sonar.login=a36fe78f64abe3b8e6db0df06a6b399c2abc2e24
```

Comment:
This is not a private token, is it?

Reply:
It is, will need to be changed when the webhook is implemented

Reply:
OK, so when the webhook is added this line can be removed?

Reply:
It will definitely need to be changed at least, I'm not positive if you will need to add your own token or not.